code/modules/VR/vr_avatar.dm

```diff
@@ -0,0 +1,87 @@
+
+
+/mob/living/carbon/human/virtual_reality
+	var/mob/living/carbon/human/real_me //The human controlling us, can be any human (including virtual ones... inception...)
```

Comment:
Yeah, that's fine.
On /tg/ we just have it use landmarks and premapped rooms.
We currently have a total of 0 premapped rooms because despite people wanting the feature, nobody really wanted to work on it, sad really.